### PR TITLE
misc fixes for f33 integration tests

### PR DESCRIPTION
**Pull request by baude**
some small fixes for testing on fedora 33 (non-btrfs)

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@containers/podman-maintainers PTAL

File: test/e2e/run_test.go [hunk 1]
```diff
@@ -261,6 +261,7 @@ var _ = Describe("Podman run", func() {
 	})
 
 	It("podman run user capabilities test", func() {
+		os.Setenv("CONTAINERS_CONF", "/dev/null")
```
Comment (vrothberg) [inline test/e2e/run_test.go:264]: A comment would be helpful

File: test/e2e/run_test.go [hunk 2]
```diff
@@ -293,6 +294,7 @@ var _ = Describe("Podman run", func() {
 	})
 
 	It("podman run user capabilities test with image", func() {
+		os.Setenv("CONTAINERS_CONF", "/dev/null")
```
Comment (vrothberg) [inline test/e2e/run_test.go:297]: Same here.

Do we have to restore the previous value of CONTAINERS_CONF?

**Review by vrothberg — COMMENTED** [review]
Changes LGTM but comments would help. Can you add some background to the commit message? I will certainly have forgotten by November why these changes were needed.

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM, but tests aren't hip?

**Pull request merged**